init.lua

```diff
@@ -585,6 +585,7 @@ require('lazy').setup {
       local ensure_installed = vim.tbl_keys(servers or {})
       vim.list_extend(ensure_installed, {
         'stylua', -- Used to format lua code
+        'markdownlint', -- Used to lint markdown
       })
```

Comment:
I think it would be great if we could have Mason install the tooling from within the nvim-lint and conform.nvim plugin sections rather than from the LSP setup function. But I'm not sure what would be a compatible approach 🤔 Does anyone have any idea?

This is how we could parse `linters_by_ft` and procure a list of tools for Mason to be installed:

```lua
local ensure_installed = {}
for filetype, linters in pairs(lint.linters_by_ft) do
  for _, linter in ipairs(linters) do
    table.insert(ensure_installed, linter)
  end
end
-- TODO: install ensure_installed with mason
```